Redirects UI text, AIW instructions, library fixes (#2137)
okta/okta-developer-docs: packages/@okta/vuepress-site/docs/guides/updating-saml-cert/revert-to-sha1/index.md

If your ISV does not accept certificates with a SHA256 signature, you can revert the settings to use the previous SHA1 certificate by rolling
over the app key to specify the SHA1 certificate you previously associated with your integration.

> #### Step 1: List your apps and get the id, name, and label for the app to revert

Reviewer: ```suggestion
#### Step 1: List your apps and get the ID, name, and label for the app to revert
```

> After you complete step 3, your users can't access the SAML app until you complete step 4.

#### Step 3: Update the key credential for the application with the SHA1 certificate

Use the [Apps API](/docs/reference/api/apps/#update-key-credential-for-application)
to update the key credential for the application to specify the kid of the SHA1 certificate that you retrieved in Step 2.

This step is the same as
[Step 3](#step-3-update-the-key-credential-for-the-app-to-specify-the-new-signing-key-id), above.

#### Step 4: Upload the SHA1 certificate to the ISV

1. In the Admin Console, go to **Applications** > **Applications**.

> 1. Click on your app integration.

Reviewer 2: ```suggestion
1. Select your app integration.
```

1. Select **Sign-On Options**.
1. Click **View Setup Instructions**, as shown below.<br/>![Accessing SAML Setup Instructions](/img/saml_setup_link.png)

> 1. Perform the setup for your app again, using the instructions provided. During this setup, you will upload the certificate in a specified format, the metadata, or the certificate fingerprint.

Reviewer: ```suggestion
1. Perform the setup for your app again by using the instructions provided. During this setup, you can upload the certificate in a specified format, the metadata, or the certificate fingerprint.
```
